### ISSUE TITLE
right tag for the right job

### ISSUE THREAD

**Issue by rimmesbe** [gary9630/prj-rev-bwfs-dasmoto#2]
We don't really want the price text to be a link, we do want it to be blue. Use a span or even better a `<strong>` tag since we want it to be default bold.

https://github.com/gary9630/prj-rev-bwfs-dasmoto/blob/master/Unit2_Dasmoto/index.html#L14

**Comment by gary9630** [comment]
Yes, I know <strong> can make it bold, but if I need to change the color in the same time, I think this line will be too long for describing the styles of it. Thanks for your comment.

**Comment by rimmesbe** [comment]
`<strong class="price">`
.price { color: blue; }

That's all you need. Font is handled by inheritance and bold handled by the tag.
